packages/mds-compliance-engine/@types/index.ts

```diff
@@ -1,5 +1,5 @@
 import { ComplianceSnapshotDomainModel } from '@mds-core/mds-compliance-service'
-import { Device, Telemetry, UUID, VehicleEvent } from '@mds-core/mds-types'
+import { Device, Device_v1_1_0, Telemetry, UUID, VehicleEvent, VehicleEvent_v1_1_0 } from '@mds-core/mds-types'
```

Comment:
nit: Might make sense to just use Device & VehicleEvent types here instead of yoinking in the 1_1_0 versions